requirements.txt

```diff
@@ -12,4 +12,5 @@ ninja
 peft>=0.8.0
 datasets>=2.15.0
 fire
+typing
```

Comment:
typing is part of the standard library from Python>=3.5  (https://docs.python.org/3/library/typing.html#typing-support-for-type-hints)

Reply:
Are you suggesting `typing` be removed from requirements?

Reply:
@seshapad 
My thinking is that the other dependencies like `transformers>=4.34.1` are tested stable for Python 3.8+. Given that it would be recommended to use Python 3.8+ which comes packed with typing as part of the standard lib. So I was just thinking should we be removing this from requirements?